client-src/elements/chromedash-feature-page.js

```diff
@@ -424,9 +439,12 @@ export class ChromedashFeaturePage extends LitElement {
         <p>
           <label>Implementation status:</label>
           <b>${this.feature.browsers.chrome.status.text}</b>
-          ${this.feature.browsers.chrome.bug ? html`
+          ${this.feature.browsers.chrome.bug ? enhanceUrl(
```

Comment:
This removed the parenthesis in the case where the bug link is enhanced.  But, that is OK because parenthesis would not look good with the badge.

Reply:
Would you like summary here also go into the tooltip? @jrobbins

Reply:
Yes.